Client/src/components/app-form/AppForm.vue

```diff
@@ -47,7 +47,7 @@ const emit = defineEmits(['validated', 'success']);
 const form = ref(null);
 const isValid = ref(false);
 const shouldValidate = ref(false);
-
+let errorMessage = ref("");
```

Comment:
Deze mag ook een `const` zijn. 

# Rule of thumb
- When using a `ref` of `reactive`, use a `const` variable.
- When you want to change a value of a variable (value and/or type) make it a `let`
- When it doesn't change, or it is an object or array that needs to be changed, make it a `const`

Reply:
huh die is bij mij gewijzigd

Reply:
![image](https://user-images.githubusercontent.com/21371624/193222716-413fdf13-1a80-40e1-8a88-a4668d686cf0.png)